kollect-core/src/main/kotlin/kollect/KollectMonadError.kt

```diff
@@ -0,0 +1,30 @@
+package kollect
+
+import arrow.Kind
+import arrow.TC
+import arrow.core.Either
+import arrow.typeclass
+import arrow.typeclasses.MonadError
+
+@typeclass(syntax = false)
+interface KollectMonadError<F> : MonadError<F, KollectError>, TC {
```

Comment:
@nomisRev had some concerns originally with this instance so maybe worth for him taking a look